staxrip@2.48.1: download via aria2 failed

console output
```console
❯ scoop update staxrip
staxrip: 2.48.0 -> 2.48.1
Updating one outdated app:
Updating 'staxrip' (2.48.0 -> 2.48.1)
Downloading new version
Starting download with aria2 ...
Download: 07/09 13:05:34 [ERROR] CUID#8 - Download aborted. URI=https://github.com/staxrip/staxrip/releases/download/v2.48.1/StaxRDownload: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/staxrip/staxrip/releases/download/v2.48.1/StaxRipDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: ddd7b9|ERR |       0B/s|C:/Scoop/cache/staxrip#2.48.1#604f2d9.7z
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://github.com/staxrip/staxrip/releases/download/v2.48.1/StaxRip-v2.48.1-x64.7z
    referer=https://github.com/staxrip/staxrip/releases/download/v2.48.1/
    dir=C:\Scoop\cache
    out=staxrip#2.48.1#604f2d9.7z

ERROR & 'C:\Scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Scoop\cache\staxrip.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.5 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=1M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=3712 --continue --summary-interval=0 --auto-save-interval=1 -l C:\Scoop\aria2log.log --max-overall-download-limit=256K

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Extras/issues/new?title=staxrip%402.48.1%3a+download+via+aria2+failed
```

--- a/bucket/staxrip.json
+++ b/bucket/staxrip.json
@@ -5,8 +5,8 @@
     "license": "MIT",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/staxrip/staxrip/releases/download/v2.48.1/StaxRip-v2.48.1-x64.7z",
-            "hash": "cb319253ece37413549710e9116e53a31c1613c081213b7dcf3ea1c3f9bc5105",
+            "url": "https://github.com/staxrip/staxrip/releases/download/v2.48.1/StaxRip-v2.48.1-x64-REPACK.7z",
+            "hash": "e4a1be2a1f6afeb55d7972786f34a18fa905cc9a1f52c752c3304d6596e14090",
             "extract_dir": "StaxRip-v2.48.1-x64"
         }
     },

